Lesson15/lesson15-ezgi/script.js

```diff
@@ -98,23 +98,23 @@ const string2Exercise13 =
 	'Bundle deal: Save $50 when you buy two products together!';
 const percentSymbol = '%';
 
-console.log('Exercise13: ');
-console.log('Exercise13: ');
+console.log('Exercise13: ',string1Exercise13.includes(percentSymbol));
+console.log('Exercise13: ',string2Exercise13.includes(percentSymbol));
 console.log('------------------------');
 // 14. Use the `substring` method to extract the word "Databases" from string stringExercise14
 // based on two indices and print it to the console.
 const stringExercise14 =
 	'Databases are an essential component of many applications.';
-
-console.log('Exercise 14: ');
+	const result2 = stringExercise14.substring(0,8);
```

Comment:
In programming, we avoid using hardcoded values as they do not create a flexible solution. If the sentence changes to, for example, 'An essential component of many applications are databases.' we will not be able to catch the word "Databases" any more.

So here you'd need to find a starting index of "Databases" word in the string and use it as a first value in the substring.
Then, find the length of the word "Databases" and combine it with starting index to find the end position.

Use 'indexOf' function (see ex. 5 above) for finding the starting index and .length for finding the length.